jbpm-wb-integration/jbpm-wb-integration-backend/src/main/java/org/jbpm/workbench/wi/backend/server/workitem/ServiceTaskServiceImpl.java

```diff
@@ -130,14 +140,19 @@ public void saveConfiguration(ServiceTasksConfiguration configuration) {
     }
 
     @Override
-    public List<String> addServiceTasks(String gav) {
-        List<String> installedServiceTasks = new ArrayList<>();
+    public Map<String, List<String>> addServiceTasks(String gav) {
         GAV actualGav = new GAV(gav);
         File uploadedServiceArtifact = m2Repository.getArtifactFileFromRepository(actualGav);
         if (uploadedServiceArtifact == null || !uploadedServiceArtifact.exists()) {
             throw new RuntimeException("No file found for artifact " + gav);
         }
-        
+        List<String> createdList = new ArrayList<>();
+        List<String> updatedList = new ArrayList<>();
+        Map<String, List<String>> resultMap = new HashMap<String, List<String>>() {{
+            put(RepoService.CREATED, createdList);
```

Comment:
please avoid this inline creation of a map.
```suggestion
            Map<String, List<String>> resultMap = new HashMap<String, List<String>>();
            resultMap.put(RepoService.CREATED, new ArrayList<>());
            resultMap.put(RepoService.UPDATED, new ArrayList<>());
```

Reply:
done